ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -50,55 +50,56 @@ open class Tooltip: NSObject, TokenizedControlInternal {
                                                            arrowMargin: tokenSet[.backgroundCornerRadius].float,
                                                            tokenSet: tokenSet
         )
-        let tooltipView = TooltipView(message: message,
-                                      title: title,
-                                      textAlignment: textAlignment,
-                                      positionController: positionController,
-                                      tokenSet: tokenSet)
-        self.tooltipView = tooltipView
-        tooltipView.accessibilityViewIsModal = true
-
-        self.onTap = onTap
-
-        self.dismissMode = UIAccessibility.isVoiceOverRunning ? .tapOnTooltip : dismissMode
-
-        let gestureView = TouchForwardingView(frame: window.bounds)
-        self.gestureView = gestureView
-        switch self.dismissMode {
-        case .tapAnywhere:
-            window.addSubview(tooltipView)
-            window.addSubview(gestureView)
-            gestureView.onTouches = { _ in
-                self.handleTapGesture()
-            }
-        case .tapOnTooltip, .tapOnTooltipOrAnchor:
-            window.addSubview(gestureView)
-            window.addSubview(tooltipView)
-            gestureView.forwardsTouches = false
-            tooltipView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTapGesture)))
-            if self.dismissMode == .tapOnTooltipOrAnchor {
-                gestureView.passthroughView = anchorView
-                gestureView.onPassthroughViewTouches = { _ in
+        self.tooltipViewController = TooltipViewController(message: message,
+                                                           title: title,
+                                                           textAlignment: textAlignment,
+                                                           positionController: positionController,
+                                                           tokenSet: tokenSet)
+        if let tooltipViewController = tooltipViewController, let tooltipView = tooltipViewController.view {
+            let rootViewController = window.rootViewController
+            rootViewController?.addChild(tooltipViewController)
```

Comment:
if we are keeping this addchild, willmove() logic, that means in dismissal of tooltip we need to clean up properly from the rootviewcontroller.

Reply:
I called tooltipViewController.willMove(toParent: nil) and tooltipViewController.removeFromParent(), what else needs to be done to clean up?

Reply:
I made sure to follow the documentation and don't see anything missing from clean up, so I'm going to resolve this comment. Let me know if you see something else I should have!